sympy/matrices/expressions/tests/test_diagonal.py

```diff
@@ -9,9 +9,18 @@
 d = DiagonalOf(X)
 
 def test_DiagonalMatrix():
+    x = MatrixSymbol('x', n, 1)
```

Comment:
Let's make that `n, n` --> when the other issue gets fixed then this test won't fail.